### ISSUE TITLE
Allow rubric grading drafts and automatically generate hw reports

### ISSUE THREAD

**Issue by MasterOdin** [Submitty/Submitty#292]
This would be a flag on the grades table that is 0 for it being a draft and 1 is submitted. Grades marked as drafts are not released to students, colored differently on summary page and should still bring student up for normal grading, but be sorted behind rubrics with no info on it.

TAs can mark a rubric as a draft at any time while grading and also rubrics would be marked as drafts if exception added to the system for a homework in case one of the automated parts is now not right.

**Comment by bdrumheller** [comment]
When questions are added, all submitted rubric grades should be converted to drafts.

**Comment by bmcutler** [comment]
A draft of this is in the rubric grading drafts branch

**Comment by MasterOdin** [comment]
We should remove the HWReport option from the menu bar once this is done (#492)

**Comment by bmcutler** [comment]
HW reports part done.   
Rubric Grading drafts is long term.

**Comment by bmcutler** [comment]
Redesign to grading by component (& submitting by component) supercedes need for grading drafts.  (May open a new issue in the future if we have a clear use case/need for drafts).
